src/main/webapp/components/sources-info/sources-info.js

```diff
@@ -80,6 +88,21 @@ export const SourcesInfo = props => {
       </Card>
     </div>
   )
+
+  if (error) {
+    return (
+      <div>
+        {SourcesDropDown}
+        <SnackbarRetry
```

Comment:
The error would be from the faceted Source Query . So if the facetedQuery would fail and the the other Source query succeeded we would show the sources along with a SnackBar retry for the facetedQuery. Since I'm going to consolidate it into one query  I will just return the SnackBarRetry here